flink-end-to-end-tests/flink-datastream-allround-test/src/main/java/org/apache/flink/streaming/tests/DataStreamAllroundTestProgram.java

```diff
@@ -113,7 +113,7 @@ public static void main(String[] args) throws Exception {
                                                                     .getInnerPayLoad()
                                                                     .getSequenceNumber()
                                                             == (event.getSequenceNumber() - 1)) {
-                                                System.out.println(
+                                                throw new Exception(
```

Comment:
I wonder if giving some information (expected actual) would help debugging. Not too sure.